openstack/identity/v3/extensions/tokens/requests.go

```diff
@@ -0,0 +1,293 @@
+package tokens
```

Comment:
I guess token request for trust should be in extensions/trust/token.
And also this file seems contain redundant code.